install_config/install/prerequisites.adoc

```diff
@@ -805,3 +805,61 @@ Following the installation process, you can configure {product-title} for
 xref:../../install_config/configuring_aws.adoc#install-config-configuring-aws[AWS],
 xref:../../install_config/configuring_openstack.adoc#install-config-configuring-openstack[OpenStack], or
 xref:../../install_config/configuring_gce.adoc#install-config-configuring-gce[GCE].
+
+[[prereq-containerized-glusterfs-considerations]]
+=== Containerized GlusterFS Considerations
+
+If you choose to configure
+xref:../../install_config/install/advanced_install.adoc#advanced-install-containerized-glusterfs-persistent-storage[containerized GlusterFS persistent storage] for your cluster, or if you choose to
+configure a
+xref:../../install_config/install/advanced_install.adoc#advanced-install-glusterfs-backed-registry[containerized GlusterFS-backed OpenShift Container Registry], you must consider the following prerequisites.
+
+[[prereq-glusterfs-storage-nodes]]
+==== Storage Nodes
+
+To use containerized GlusterFS persistent storage:
+
+- Each storage node is recommended to have at least 1 raw block device with least
+100 GB available.
+
+To run a containerized GlusterFS-backed OpenShift Container Registry:
+
+- A minimum of 3 storage nodes must be available for 3-way replication.
+- Each storage node must have at least 1 raw block device with at least 10 GB of
+free storage. 
```

Comment:
For the 10ish lines above s/the number/the words for the number.

IBM p156 about number words.